downloads/swan-lake-release-notes/swan-lake-2201.12.0/RELEASE_NOTE.md

```diff
@@ -58,6 +58,38 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 12 (2201.12.0)
 
 ### New features
 
+#### Consolidate-packages tool
+
+- Introduced the `consolidated-packages` bal tool to generate code required to consolidate multiple Ballerina services into a single executable.
+- Simplifies service consolidation to enable monolith-style deployments.
+
+```toml
+[package]
+org = "myorg"
+name = "myapp"
+version = "0.1.0"
+
+[[tool.consolidate-packages]]
+id = "consolidateSvc"
+options.services = ["myorg/menu_service", "myorg/order_service"]
+```
+
+Automation tools and CI/CD pipelines can integrate the CLI tool to automatically generate a Ballerina package containing the consolidated services.
+
+To install the tool, run:
+
+```bash
```

Comment:
```suggestion
```
```

Reply:
Checked the U11 release note. It didn't have command outputs as bullet points. Do we need to add it?

Anyway, found some inconsistent indentations. I have fixed them.